spacy/tests/vocab_vectors/test_lexeme.py

```diff
@@ -22,16 +22,21 @@ def test_issue600():
     doc[0].tag_ = "NN"
 
 
-@pytest.mark.parametrize("text1,prob1,text2,prob2", [("NOUN", -1, "opera", -2)])
-def test_vocab_lexeme_lt(en_vocab, text1, text2, prob1, prob2):
-    """More frequent is l.t. less frequent"""
-    lex1 = en_vocab[text1]
-    lex1.prob = prob1
-    lex2 = en_vocab[text2]
-    lex2.prob = prob2
-
-    assert lex1 < lex2
-    assert lex2 > lex1
+@pytest.mark.parametrize("word1,prob1,word2,prob2", [("NOUN", -1, "opera", -2)])
```

Comment:
The original test did not make sense. `lex1 < lex2` is comparing `ORTH` and not `PROB`. Replaced with a test for the lookups table creation.